daemon/delete.go

```diff
@@ -79,7 +79,12 @@ func (daemon *Daemon) rmLink(container *container.Container, name string) error
 func (daemon *Daemon) cleanupContainer(container *container.Container, forceRemove, removeVolume bool) (err error) {
 	if container.IsRunning() {
 		if !forceRemove {
-			err := fmt.Errorf("You cannot remove a running container %s. Stop the container before attempting removal or use -f", container.ID)
+			state := container.StateString()
+			procedure := "Stop the container before attempting removal or use -f"
+			if state == "paused" {
+				procedure = "Unpause and stop the container before attempting removal"
+			}
```

Comment:
After unpausing, the container can be force deleted as well in addition to stopping and removing. I think the above message needs to capture that as well. Thanks